WordPressAuthenticatorTests/Logging/LoggingTests.swift

```diff
@@ -0,0 +1,70 @@
+import XCTest
+
+@testable import WordPressAuthenticator
+
+private class CaptureLogs: NSObject, WordPressLoggingDelegate {
+    var verboseLogs = [String]()
+    var debugLogs = [String]()
+    var infoLogs = [String]()
+    var warningLogs = [String]()
+    var errorLogs = [String]()
```

Comment:
Same nitpick as [in WordPressKit-iOS](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/550#discussion_r1033261108):

```suggestion
    private(set) var verboseLogs = [String]()
    private(set) var debugLogs = [String]()
    private(set) var infoLogs = [String]()
    private(set) var warningLogs = [String]()
    private(set) var errorLogs = [String]()
```

Reply:
Thanks! I've taken your code suggestion.